src/main/java/org/jabref/gui/desktop/JabRefDesktop.java

```diff
@@ -131,7 +131,7 @@ private static void openDoi(String doi) throws IOException {
 
     public static void openCustomDoi(String link, PreferencesService preferences, DialogService dialogService) {
             DOI.parse(link)
-                        .map(identifier -> (DOI) identifier)
+                        .map(identifier -> identifier)
```

Comment:
?

Reply:
seems to be have done automatically by eclipe